src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPECIAL_DELIMITER = "!Delimiter@";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is better to break down the logic of the program 3 methods:
1. Reading from file
2. Report creation
3. Writing the report to a file

Reply:
There should only be logic in this method:
String data = readFromFile(fromFileName);
String report = createReport(data);
writeToFile(toFileName, report);